src/js/claims-status/reducers/appeals.js

```diff
@@ -4,27 +4,65 @@ import {
   SET_APPEALS,
   SET_APPEALS_UNAVAILABLE,
   FETCH_APPEALS_PENDING,
-  FETCH_APPEALS_SUCCESS
+  FETCH_APPEALS_SUCCESS,
 } from '../actions/index.jsx';
 
+import {
+  USER_FORBIDDEN,
+  RECORD_NOT_FOUND,
+  VALIDATION_ERROR,
+  BACKEND_SERVICE_ERROR,
+  FETCH_APPEALS_ERROR,
+  AVAILABLE
+} from '../utils/appeals-v2-helpers';
+
 const initialState = {
   available: true,
+  availabilityError: null,
 };
 
 export default function appealsReducer(state = initialState, action) {
   switch (action.type) {
     case FETCH_APPEALS_PENDING:
       return _.set('appealsLoading', true, state);
-    case FETCH_APPEALS_SUCCESS:
+    case FETCH_APPEALS_SUCCESS: // Appeals v1 and v2
+      // appeals are added to Redux store via FETCH_APPEALS_SUCCESS
+      // reducer in claims-list.js
       return _.merge(state, {
         appealsLoading: false,
         available: true,
+        v2Availability: AVAILABLE // New and improved! More bits of info!
```

Comment:
Oof. Can we add in a `TODO` to clean this up when we go full v2? This is...less than desirable, but I understand why you'd do it this way.

I'm really wondering if we should just make a completely separate reducer for v2, actually. :thinking: Separation of concerns and all that.